webapp/src/ts/components/analytics-targets-progress/analytics-targets-progress.component.html

```diff
@@ -8,11 +8,11 @@
   </div>
 
   <div class="goal" *ngIf="target?.goal > 0" [ngStyle]="{ 'padding-left': (target?.goal || 0) + '%' }">
-    <label [ngClass]="{ 'pin-left': !aggregate && target?.goal < 15, 'pin-right': aggregate || target?.goal > 85 }">
+    <label [ngClass]="{ 'pin-left': !aggregate && target?.goal < 10, 'pin-right': aggregate || target?.goal > 85 }">
       <span>{{'analytics.target.monthly_goal' | translate}}</span>
       {{ target.goal | localizeNumber }}<span *ngIf="target?.type === 'percent'">%</span>
     </label>
-    <div class="bar" *ngIf="target?.type === 'percent'"></div>
+    <div class="bar" *ngIf="target?.type === 'percent'" [ngStyle]="{ 'margin-left': (aggregate || target?.goal > 95) ? '-10px' : '-2px' }"></div>
```

Comment:
The 100% goal's bar isn't going all the way to the right.

<img width="492" alt="Screenshot 2023-11-10 at 10 31 38 am" src="https://github.com/medic/cht-core/assets/66472237/873b56c7-e8ac-4970-bbbc-2ef868c14c34">

Instaed of this ngstyle, please add a class

```
<div class="bar" *ngIf="target?.type === 'percent'" [class.big-goal]="<<condition>>"></div>
```
So it has a default margin, but when this class is active then the class apply a new margin for that case only. It's a bit cleaner

Reply:
Thanks @latin-panda for the suggestion. I have extracted the instyle to a class. 

However, the 100% goal is all correct on my side here are the screenshots of the different percentages I have tested
<img width="319" alt="Screenshot 2023-11-10 at 08 21 13" src="https://github.com/medic/cht-core/assets/2597305/c0933bcd-0998-4eb5-947e-034c6bc59645">
<img width="309" alt="Screenshot 2023-11-10 at 08 17 05" src="https://github.com/medic/cht-core/assets/2597305/ecaf4cd3-a8a5-4c49-9335-27b3dd94fa5e">
<img width="321" alt="Screenshot 2023-11-10 at 08 19 11" src="https://github.com/medic/cht-core/assets/2597305/c5ce3ef8-b15d-46eb-8ea1-e571d4172ce6">
<img width="325" alt="Screenshot 2023-11-10 at 08 20 39" src="https://github.com/medic/cht-core/assets/2597305/f3e89f3f-ead1-4339-86ea-82242700b247">

Reply:
This is when testing the progress bar in the aggregate targets. Can you please check it?

Reply:
I have seen it, it's because I changed `L116` in `theme.less`. 

Deleting `L192` in `targets.less` and letting the margin of `.progress` be styled with `L116` in `theme.less` fixes the issue. Now both aggregate targets and targets are correct. 
<img width="509" alt="Screenshot 2023-11-10 at 09 07 30" src="https://github.com/medic/cht-core/assets/2597305/9120d504-8f47-4f52-9016-52c6e67e5976">